include/bluetooth/mesh/gen_prop_srv.rst

```diff
@@ -3,59 +3,61 @@
 Generic Property Servers
 ########################
 
-A Generic Property Server holds a list of Device Properties.
-There are four different property servers:
+A Generic Property Server holds a list of device properties.
+The following different property servers are available:
```

Comment:
This change was a bit awkward I think. I get the motivation behind removing the count, but in this case, it's very unlikely that it'll ever change. I think the old text was better, but perhaps removing "different" would be an okay middle ground?